algobot/data.py

```diff
@@ -521,19 +522,20 @@ def get_interval_minutes(self) -> int:
         else:
             raise ValueError("Invalid interval.", 4)
 
-    def create_folders_and_change_path(self, folderName: str):
+    def create_folders_and_change_path(self, folder_name: str):
```

Comment:
Nice change! We should really convert camel case to snake case over time

Reply:
snake case is the python way. If you're happy with something we could use black?

https://black.readthedocs.io/en/stable/

We can configure it as a pre-commit hook even :)

Reply:
pylint will help enforce it ;)

Reply:
Yeap. We should really start leveraging pylint and mypy